tests/components/switcher_kis/test_button.py

```diff
@@ -0,0 +1,150 @@
+"""Tests for Switcher button platform."""
+from unittest.mock import ANY, patch
+
+from aioswitcher.api import DeviceState, SwitcherBaseResponse, ThermostatSwing
+import pytest
+
+from homeassistant.components.button import DOMAIN as BUTTON_DOMAIN, SERVICE_PRESS
+from homeassistant.const import ATTR_ENTITY_ID, STATE_UNAVAILABLE
+from homeassistant.core import HomeAssistant
+from homeassistant.exceptions import HomeAssistantError
+from homeassistant.util import slugify
+
+from . import init_integration
+from .consts import DUMMY_THERMOSTAT_DEVICE as DEVICE
+
+BASE_ENTITY_ID = f"{BUTTON_DOMAIN}.{slugify(DEVICE.name)}"
+ASSUME_ON_EID = BASE_ENTITY_ID + "_assume_on"
+ASSUME_OFF_EID = BASE_ENTITY_ID + "_assume_off"
+SWING_ON_EID = BASE_ENTITY_ID + "_vertical_swing_on"
+SWING_OFF_EID = BASE_ENTITY_ID + "_vertical_swing_off"
+
+
+@pytest.mark.parametrize("mock_bridge", [[DEVICE]], indirect=True)
+async def test_assume_button(hass: HomeAssistant, mock_bridge, mock_api):
```

Comment:
Considering both buttons act alike, maybe it would be enough to parameterize and verify that the calls are taking place to avoid duplicate code? That is just a suggestion though, IMO this PR looks good to go from my view :+1:

Reply:
I will add this in a new PR 👍